azext_edge/edge/providers/orchestration/resource_map.py

```diff
@@ -146,10 +146,11 @@ def build_tree(self, include_dependencies: bool = False, category_color: str = "
             aio_ext_obj = self.connected_cluster.get_extensions_by_type(IOT_OPS_EXTENSION_TYPE).get(
                 IOT_OPS_EXTENSION_TYPE, {}
             )
-            aio_ext_id: str = aio_ext_obj.get("id", "")
-            aio_ext = next((_ for _ in self.extensions if _.resource_id.lower() == aio_ext_id.lower()), None)
-            if aio_ext:
-                extensions_node.add(aio_ext.display_name)
+            if aio_ext_obj:
+                aio_ext_id: str = aio_ext_obj.get("id", "")
+                aio_ext = next((_ for _ in self.extensions if _.resource_id.lower() == aio_ext_id.lower()), None)
```

Comment:
nit: would really like the _ to be something like e... feels very weird to do property of wild card